resources/assets/components/Post/index.js

```diff
@@ -38,6 +38,8 @@ class Post extends React.Component {
         const newState = {...prevState};
 
         newState.loading = false;
+        // Add a cache-busting string to the end of the image url
+        // so that it changes and triggers a re-render.
         newState.post.media.url = `${json.url}?time=${new Date()}`;
```

Comment:
We'll have to think about how to handle this more generally when the [caching updates](https://github.com/DoSomething/devops/issues/321#issuecomment-329545441) kick in (since anyone else re-loading the same image URL would get the cached Fastly response then) We could either include a cache tag for each image and tell Fastly to clear it when we make a change, or maybe by always include a string like this in the image URL based on `updated_at` timestamp.

Either way, definitely not something we need to handle right now!

Reply:
For sure. Will make a note to test this out when the caching updates are made